merlin/models/tf/outputs/contrastive.py

```diff
@@ -132,6 +147,7 @@ def __init__(
         query_name: str = "query",
         candidate_name: str = "candidate",
         store_negative_ids: bool = False,
+        logq_sampling_correction: Optional[bool] = False,
```

Comment:
What's the reasoning behind making this false by default? Are there down-sides of doing LogQ correction?